public/sig-blk-en.svg

```diff
@@ -0,0 +1,8 @@
+</svg>

```

Comment:
I know we're not using it yet, but can you add the `sig-blk-fr.svg` as well? (the French text comes first in the other svg).